main.nf

```diff
@@ -43,10 +49,10 @@ process download_fastqs {
     time { 5.hour * task.attempt }
     memory { 20.GB * task.attempt }
 
-    errorStrategy { task.attempt<=10 ? 'retry' : 'finish' } 
+    errorStrategy { task.attempt<=10 & task.exitStatus != 4 ? 'retry' : 'finish' } 
```

Comment:
Same question with droplet exit code 4 ? In Smart-seq it was 2 ?

Reply:
If you look at the process code below, you'll see I've set the exit status of 4 for cases when the files are not found locally, and when data are controlled access (so local files a required). This is specific to this workflow- the smartseq one is different.